trymito.io/styles/[slug].module.css

```diff
@@ -102,11 +124,75 @@
   border-bottom: 1.5px solid black; 
 }
 
+.post_toc {
+    width: 265px;
+    min-width: 265px;
+    height: fit-content;
+    position: sticky;
+    top: 67px;
+    margin-left: 30px;
+}
+
 .post_content table thead {
     background-color: black;
     border-radius: 1rem;
 }
 
+.post_cta {
+    width: 265px;
+    min-width: 265px;
+    height: fit-content;
+    position: sticky;
+    top: 100px;
+    right: 0;
+    background-color: #F2C94C;
+    border-radius: 15px;
+    padding: 30px 30px 0 30px;
+    margin-right: 30px;
+    margin-top: 30px;
+}
+
+.suggested_blogs_section {
+    margin: 60px;
+    display: flex;
+    flex-direction: column;
+}
+
+.suggested_blogs_section h1 {
+    margin: 30px;
+    color: var(--color-light-purple);
+}
+
+.suggested_blogs_links {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.suggested_blog_card {
+    width: 250px;
+    margin: 30px;
+}
+
+@media only screen and (max-width: 1400px) {
```

Comment:
Instead of using px here, I think it's best practice to use em. That's because things like the font on our website are based on em which means when the user zooms in on their browser, the font can update correctly. But to keep everything in sync, we should use em here too.

Clearly, not the best answer in the world. I haven't touched up on my rem, em, px knowledge in a few years.